lib/loggability/log_device.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+# vim: set nosta noet ts=4 sw=4:
+# encoding: utf-8
+
+require 'loggability' unless defined?( Loggability )
+
+
+### Parent class of all Loggability's log devices and can require
+### and instantiate its subclassses by name and args
+class Loggability::LogDevice
+
+	### Parses out the target class name and its arguments from the +target_spec+
+	### then requires the subclass and instantiates it by passing the arguments
+	### The +target_spec+ comes from a config file in the format of:
+	### logging:
+	###     datadog[data_dog_api_key] (color)
+	### In the above example:
+	### 	"datadog" is the log device to send logs to
+	### 	"[data_dog_api_key]" is the argument that will be passed onto the datadog subclass
```

Comment:
This is a little unclear to me - what do you think about these changes?
`"data_dog_api_key" is the argument that will be passed onto the datadog log device's constructor`

(Hash part of the string isn't actually provided, and making it clearer what it's passed into)